forge-gui/res/editions/Warhammer 40,000 Commander.txt

```diff
@@ -15,325 +15,165 @@ ScryfallCode=40K
-29★ R Arco-Flagellant @Games Workshop
 30 R Biotransference @Anthony Devine
-30★ R Biotransference @Anthony Devine
 31 R Blight Grenade @Alexey Kruglov
-31★ R Blight Grenade @Alexey Kruglov
 32 R Chronomancer @Alexey Kruglov
-32★ R Chronomancer @Alexey Kruglov
 33 R Cryptek @David Álvarez
-33★ R Cryptek @David Álvarez
 34 U Flayed One @David Sondered
-34★ U Flayed One @David Sondered
 35 R Great Unclean One @Helge C. Balzer
-35★ R Great Unclean One @Helge C. Balzer
 36 U Hexmark Destroyer @Wonchun Choi
-36★ U Hexmark Destroyer @Wonchun Choi
 37 R Illuminor Szeras @Darren Tan
-37★ R Illuminor Szeras @Darren Tan
 38 R Lokhust Heavy Destroyer @Adrián Rodríguez Pérez
-38★ R Lokhust Heavy Destroyer @Adrián Rodríguez Pérez
 39 R Lychguard @Games Workshop
-39★ R Lychguard @Games Workshop
 40 R Mandate of Abaddon @Games Workshop
-40★ R Mandate of Abaddon @Games Workshop
 41 R Mortarion, Daemon Primarch @Helge C. Balzer
-41★ R Mortarion, Daemon Primarch @Helge C. Balzer
 42 R Necron Deathmark @Games Workshop
-42★ R Necron Deathmark @Games Workshop
 43 R Necron Overlord @David Sondered
-43★ R Necron Overlord @David Sondered
 44 R Nurgle's Conscription @Ørjan Ruttenborg Svendsen
-44★ R Nurgle's Conscription @Ørjan Ruttenborg Svendsen
 45 U Nurgle's Rot @Roman Tishenin
-45★ U Nurgle's Rot @Roman Tishenin
 46 R Out of the Tombs @Games Workshop
-46★ R Out of the Tombs @Games Workshop
 47 R Plague Drone @Games Workshop
-47★ R Plague Drone @Games Workshop
 48 U Plasmancer @Bartek Fedyczak
-48★ U Plasmancer @Bartek Fedyczak
 49 R Poxwalkers @Games Workshop
-49★ R Poxwalkers @Games Workshop
 50 R Primaris Eliminator @Logan Feliciano
-50★ R Primaris Eliminator @Logan Feliciano
 51 U Psychomancer @Alex Konstad
-51★ U Psychomancer @Alex Konstad
 52 R Royal Warden @Artur Nakhodkin
-52★ R Royal Warden @Artur Nakhodkin
 53 U Sanguinary Priest @Games Workshop
-53★ U Sanguinary Priest @Games Workshop
 54 U Sautekh Immortal @Games Workshop
-54★ U Sautekh Immortal @Games Workshop
 55 R Shard of the Nightbringer @Pierre Loyvet
-55★ R Shard of the Nightbringer @Pierre Loyvet
 56 R Shard of the Void Dragon @Alex Konstad
-56★ R Shard of the Void Dragon @Alex Konstad
 57 U Skorpekh Destroyer @Games Workshop
-57★ U Skorpekh Destroyer @Games Workshop
 58 R Skorpekh Lord @Adrián Rodríguez Pérez
-58★ R Skorpekh Lord @Adrián Rodríguez Pérez
 59 R Sloppity Bilepiper @Ørjan Ruttenborg Svendsen
-59★ R Sloppity Bilepiper @Ørjan Ruttenborg Svendsen
 60 R Tallyman of Nurgle @Games Workshop
-60★ R Tallyman of Nurgle @Games Workshop
 61 R Technomancer @David Álvarez
-61★ R Technomancer @David Álvarez
 62 R Their Name Is Death @Evan Shipard
-62★ R Their Name Is Death @Evan Shipard
 63 R Their Number Is Legion @Games Workshop
-63★ R Their Number Is Legion @Games Workshop
 64 R Tomb Blade @Darren Tan
-64★ R Tomb Blade @Darren Tan
-65 R Trazyn the Infinite @Michał Miłkowski
-65★ R Trazyn the Infinite @Michał Miłkowski
+65 R Trazyn the Infinite @Micha Mi?kowski
```

Comment:
Can you revert card 65 as the artist shouldn't be changed (your edit also contains a ?)